scripts/diagnose.sh

```diff
@@ -147,16 +147,18 @@ commands=(
 	'sysctl -a'
 	'systemctl list-units --failed --no-pager'
 	'top -b -n 1'
-	'grep -vE \"/var/cache/ldconfig/aux-cache|md5sum|/etc/hostname|/etc/machine-id|/etc/resin-supervisor/supervisor.conf|/etc/systemd/timesyncd.conf|/home/root/.rnd\" /resinos.fingerprint | md5sum --quiet -c ' # https://github.com/balena-os/meta-balena/issues/1618
+	'grep -vE \"/var/cache/ldconfig/aux-cache|md5sum|/etc/hostname|/etc/machine-id|/etc/balena-supervisor/supervisor.conf|/etc/resin-supervisor/supervisor.conf|/etc/systemd/timesyncd.conf|/home/root/.rnd\" /resinos.fingerprint | md5sum --quiet -c ' # https://github.com/balena-os/meta-balena/issues/1618
 
 	# SUPERVISOR specific commands
 	'echo === SUPERVISOR ==='
+	'$ENG exec balena_supervisor cat /etc/resolv.conf'
```

Comment:
```suggestion
	'$ENG exec $($ENG ps | awk '/supervisor/{print $1}') cat /etc/resolv.conf'
```

maybe?

Reply:
fwiw there's probably a cleaner way that excludes awk and uses `--filter` and `--format` engine flags, but this is pretty portable

Reply:
```suggestion
	'$ENG exec $($ENG ps --filter "name=resin_supervisor" --filter "name=balena_supervisor" -q) cat /etc/resolv.conf'
```

looks ok

Reply:
you could ALSO do:

```
	'$ENG exec $($ENG ps --filter "name=_supervisor" -q) cat /etc/resolv.conf'
```

but i don't like that as much since it's a substring match and could be overloaded by a user container